index.js

```diff
@@ -56,5 +56,5 @@ module.exports = {
       },
     },
   },
-  html: '<label for="{{quote.id}}">{{quote.label}}</label><input type="{{quote.type}}" id="{{quote.id}}" name="{{quote.name}}" value="{{quote.value}}" placeholder="{{quote.placeholder}}" /><label for="{{author.id}}">{{author.label}}</label><input type="{{author.type}}" id="{{author.id}}" name="{{author.name}}" value="{{author.value}}" placeholder="{{author.placeholder}}" /><label for="{{source.id}}">{{source.label}}</label><input type="{{source.type}}" id="{{source.id}}" name="{{source.name}}" value="{{source.value}}" placeholder="{{source.placeholder}}" />',
+  html: '<div class="form--field"><label for="{{quote.id}}">{{quote.label}}</label><input type="{{quote.type}}" id="{{quote.id}}" name="{{quote.name}}" value="{{quote.value}}" placeholder="{{quote.placeholder}}" /></div><div class="form--field"><label for="{{author.id}}">{{author.label}}</label><input type="{{author.type}}" id="{{author.id}}" name="{{author.name}}" value="{{author.value}}" placeholder="{{author.placeholder}}" /></div><div class="form--field"><label for="{{source.id}}">{{source.label}}</label><input type="{{source.type}}" id="{{source.id}}" name="{{source.name}}" value="{{source.value}}" placeholder="{{source.placeholder}}" /></div>',
```

Comment:
Oh, I see, so regardless of them having one or more-than-one input, you need this wrapped around _each_ input/label combo? I am wrong, that is not something we can do in html.js. We want to avoid blanket manipulation of the _internal_ html from a plugin.

So, that requirement will make input plugins not created by us no fit in with our visual styling.

Is there no other way to do this? Say by giving the inputs styling?